languages/common-lisp/exercises/concept/arithmetic/.docs/introduction.md

```diff
@@ -0,0 +1,50 @@
+## Numerical Types
+
+In Common Lisp, like many languages, numbers come in a couple of types. This
+exercise focuses on getting to grips with two of the most basic number-types:
+- Integers: whole numbers without a decimal point (like `-6`, `0`, `25`, `1234`,
+  etc.)
+- Floating-Point Numbers: fractional or whole numbers including a decimal point (like
+  `3.14`, `-1.7`, `99.99`, `2048.0`)
+
+In general, if you are working with only whole numbers, you should prefer
+integers as they don't suffer from the same loss of precision as floating-point
+numbers do over many calculations.
+
+## Arithmetic & Prefix Notation
+
+Common Lisp uses the standard arithmetic operators for most operations (you can
+find a [full list here](http://clhs.lisp.se/Body/12_aa.htm)), but is somewhat
+unique in using a "prefix-notation" as opposed to the more familiar
+"infix-notion". More visually:
+
+```lisp
+;; Infix-notation (non-lisp languages)
+1 + 2 + 3 + 4 + 5 ; => 15
+;; Prefix-notation (lisp languages)
+(+ 1 2 3 4 5) ; => 15
+```
+
+While prefix notion turns some operations like `2 + 2` into the somewhat
+unfamiliar `(+ 2 2)` form, it makes it much easier to operate on more than one
+number at a time.
```

Comment:
I think a quick example here of 'more than one number' would be good, in case the reader cannot easily imaging `(+ 1 2 3 4 5 6 7 8 9)` as a thing one could do in a programming language.

Reply:
I showed that in the code snippet above under prefix notation. Do you think I should highlight that a bit more?

Reply:
Hmm if it was already there then i just missed it. nevermind.